aiida_vasp/utils/vasp/ibrion.py

```diff
@@ -0,0 +1,106 @@
+"""Utilities to decide on appropriate value for IBRION tag and catch clashes with other tags."""
+import enum
+
+from aiida.common.utils import classproperty
+from aiida_vasp.utils.vasp.incar_param import AbstractIncarParam
+
+
+class IbrionFlags(enum.Enum):
+    """Enum with more descriptive names for the various possible IBRION values."""
+    NO_UPDATE = -1
+    MOLECULAR_DYNAMICS = 0
+    IONIC_RELAXATION_RMM_DIIS = 1
+    IONIC_RELAXATION_CG = 2
+    IONIC_RELAXATION_DAMPED_MD = 3
+    HESSIAN_FD_FULL = 5
+    HESSIAN_FD_SYMMETRIZED = 6
+    HESSIAN_PT_FULL = 7
+    HESSIAN_PT_SYMMETRIZED = 8
+    IMPROVED_DIMER_METHOD = 44
+
+    @classproperty
+    def ionic_relaxation_options(self):
+        return [self.IONIC_RELAXATION_RMM_DIIS, self.IONIC_RELAXATION_CG, self.IONIC_RELAXATION_DAMPED_MD]
+
+
+class Ibrion(AbstractIncarParam):
```

Comment:
Actually python has the following conventions:

```python
ClassName
GLOBAL_VARIABLE_NAME
local_variable_name
function_name
method_name
```

I feel like we need a good reason to break these, considering that they are currently enforced by pre-commit. Straying from these conventions would decrease readability and raise the barrier to entry for new devs.

Reply:
Yes. Also, I think that the specifications are for CapWords, so if you do, CW then you should be fine? A bunch of these tags are indeed of that type. I am fine with either, so since you started with this, let us keep it. My eyes hurt, but I will put on my sunglasses.